### DISCUSSION TITLE
1 Billion Row Challenge and QSV

### DISCUSSION THREAD

**Discussion by ramkumarkb** [dathere/qsv#1564] [General]
https://www.morling.dev/blog/one-billion-row-challenge/ 

It wwould be really nice to have a blog or some shout out on trying 1brc with QSV. 

There was an excellent blog post on trying Rust for 1BRC - https://curiouscoding.nl/posts/1brc/

**Comment by aborruso** [comment]
Another related interesting post https://ibis-project.org/posts/1brc/

**Comment by ramkumarkb** [comment]
https://aminediro.com/posts/billion_row/ was this featured in [T-W-I-R](https://this-week-in-rust.org/blog/2024/01/24/this-week-in-rust-531/) today

**Comment by jqnatividad** [comment]
Challenge accepted @ramkumarkb @aborruso !

As qsv's top Goal is Speed, it's only appropriate to step up to the challenge! 😄

**Comment by jqnatividad** [comment]
Update:
Held off on running the benchmark till the next release - 0.123.0, which is happening over the weekend...

**Reply by aborruso** [reply]
> Held off on running the benchmark till the next release - 0.123.0, which is happening over the weekend...

So there will be important news in this release 🙂

**Comment by ondohotola** [comment]
Is there a simple method to get the dataset as a csv.gz ?

**Comment by jqnatividad** [comment]
Now that 0.123.0 is released, will see if I can find a simple way to run this challenge....
